dev.php

```diff
@@ -103,9 +103,12 @@
 					'label' => 'Cite',
 					'attr'  => 'source',
 					'type'  => 'text',
-					'placeholder' => 'Test placeholder',
+					'customAttributes' => array(
```

Comment:
Renamed to meta. I think that an attributes property is already used by backbone so don't want to cause confusion.